libs/ledger-live-common/src/families/near/hw-app-near.ts

```diff
@@ -0,0 +1,96 @@
+import type Transport from "@ledgerhq/hw-transport";
```

Comment:
Reasons:
- Less room for error back when we were using `yalc`
- Wasn't aware that there was a preferred architecture - happy to change this if necessary (maybe in a follow-up PR since this one is already pretty big?)